<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatreonDialog</class>
 <widget class="QDialog" name="PatreonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Become a Patron</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Thanks for using Tiled! Please consider supporting further development with a small monthly donation.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="gotoPatreon">
       <property name="text">
        <string>Visit https://www.patreon.com/bjorn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="alreadyPatron">
       <property name="text">
        <string>I'm already a supporter!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maybeLaterButton">
       <property name="text">
        <string>Maybe later</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
